app/assets/scripts/map.js

```diff
@@ -80,7 +80,7 @@ function displayPhoneNumbers(descriptors) {
   var PHONE_NUMBER_LENGTH = 12;
   for (desc of descriptors) {
     // skip option descriptors
-    if (desc.value.replace) {
+    if (desc.value.replace!=null) {
```

Comment:
i was getting a frontend error when i didn't do the explicit null check -- not sure why?